epuck_ros2_driver/test/test_driver.py

```diff
@@ -34,6 +34,10 @@
 SENSORS_SIZE = 47
 ACTUATOR_SIZE = 20
 DISTANCE_FROM_CENTER = 0.035
+READ_WRITE_RETRY_N = 6
```

Comment:
I would call it `READ_WRITE_RETRY_NUMBER` (we always try to avoid abbreviations).
The same comment applies to `MESSAGE_SEND_RETRY_N`.